File: mobile/src/main/java/com/chernowii/camcontrol/camera/goproAPI/model/media/GoProMetadata.java
package com.chernowii.camcontrol.camera.goproAPI.model.media;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class GoProMetadata {

    @SerializedName("dur")
    public Integer duration;

    @SerializedName("profile")
    public Integer profile;

    @SerializedName("tag_count")
    public Integer tag_count;

    @SerializedName("tags")
    public List<Integer> tags = new ArrayList();

}

File: mobile/src/main/java/com/chernowii/camcontrol/camera/goproAPI/model/media/GoProMetadata.kt
package com.chernowii.camcontrol.camera.goproAPI.model.media

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class GoProMetadata {

    @SerializedName("dur")
    var duration: Int? = null

    @SerializedName("profile")
    var profile: Int? = null

    @SerializedName("tag_count")
    var tag_count: Int? = null

    @SerializedName("tags")
    var tags: List<Int> = ArrayList()

}
